.circleci/config.yml

```diff
@@ -12,10 +24,65 @@ jobs:
           command: |
             docker build -t mozilla/sops .
             docker tag mozilla/sops "mozilla/sops:$CIRCLE_SHA1"
+
+  push:
+    machine: true
+    resource_class: large
+    steps:
+      - checkout
+      - run: 
+          name: semver check
+          command: |
+            curl "https://raw.githubusercontent.com/fsaintjacques/semver-tool/d66113a6803dbffca2a3830e207a6795a8516cd1/src/semver" > semver.bash
```

Comment:
Pulling in an external tool for this feels a bit heavy weight. Can this be changed to just using `cut`?

Reply:
I agree with the concern, I locked into a hash to avoid surprises but I understand the reluctance.

1. We could commit the referenced bash script into source.  
1. We could re-write it in bash ourselves (dislike this)
1. if this project won't ever use classififers and just 1.2.3. I could do some sed manipulation.

Reply:
Updated to use cut, push step will ignore missing minor/patch versions, but requires major version.